code/src/Album.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Artist } from "./Artist";
+import { Image } from "./Image";
+import { AlbumArt } from "./AlbumArt";
+import { Controls } from "./Controls";
+import "./css/album.css";
+
+export const Album = props => {
+  return (
+    <div className="album">
+      <AlbumArt>
+        <Image title={props.name} images={props.images} />
+        <Controls playUrl={props.url} />
+      </AlbumArt>
+
+      <h2 className="album-name">
+        <a href={props.url}>{props.name}</a>
+      </h2>
+      {props.artists.map((artist, i) => {
+        return (
+          <React.Fragment key={artist.id}>
+            {i > 0 && ", "}
```

Comment:
Nice solution. I think CSS would've been a better place for this type of presentation, though. You could use the `:after` pseudo-element to add a comma, then use `:last-child` to set the content on the final element to be '' (so no comma)